src/en/reference-status.md

```diff
@@ -0,0 +1,28 @@
+
+
+# Unit status
+
+As of version 1.24 of Juju, a unit can report its status as one of the
+following:
+
+| Status       | colour in GUI* | Meaning                                      |
+| :----------- | :------------- | :------------------------------------------- |
+| blocked      | blue           | This unit needs manual intervention because of a problem associated directly with the unit itself. This may be due to bad or inconsistent configuration of the service, disk space being unavailable for the unit, or missing relations which are essential to service operations. The message should tell a human administrator what is required to unblock the service.<br> For example: “Please provide an object store. Please configure the minimum speed required by users of this service. Please link this service to a database like MySQL or Postgres.”|
+|maintenance  |yellow          |The unit is not yet providing services, but is actively doing stuff in preparation for providing those services. This is a “spinning” state, not an error state. It reflects activity on the unit itself, not on peers or related units. The Juju agent will set the unit to maintenance when it begins the process of installing the charm - downloading it, unpacking it and calling the install hook.<br>The unit may transition to the maintenance state at any time, in any hook. The maintenance state implies that the unit has no reason to think that it will not provide services soon. For example, the software might be formatting block devices, or replicating data from peer units, or handling an action which for whatever reason means that it is not actively providing service, but it is not aware of anything on its own config or in related unit config that will prevent it from bringing up its services.<br>Counterparts to this unit should expect that service calls to this unit will fail during the period it is in the maintenance state, but unless they have an urgent need to move to another counterpart, they should just wait for this unit to finish churning.<br> Being in maintenance does not require human intervention.|
+|waiting      |yellow         | The unit is unable to progress to an active state because a service to which it is related is not running. That service might be in error, blocked, waiting or maintenance. This unit expects it will be able to proceed towards active as soon as the things it is concerned about are resolved there - it needs no human attention (otherwise its status would be blocked).|
+|active |green| This unit believes it is correctly offering all the services it is primarily installed to provide. A service can be active even if additional relations are being configured and setup. For example, a ceph osd service that is fully connected up to its controllers is active, even if you then relate it to nagios; setting up the monitoring subordinate does not impinge on the ceph osd functionality which is its primary purpose for existing.|
```

Comment:
Ceph OSD (2 places), or just "OSD" (remove "ceph") for the second instance
Nagios

"fully connected up to" - odd. Maybe "fully joined with", or just "joined with"

"[Ceph OSD] controllers" - I'm not sure what this refers to. Once set up, an OSD services Ceph client requests. Is this referring to bringing up an OSD? If so, it is probably referring to its "Monitors".

Personally, "setup" is a noun and "set up" is a verb. Same for "login" and "log in".